test/hotspot/jtreg/compiler/gcbarriers/UnsafeIntrinsicsTest.java

```diff
@@ -22,7 +22,7 @@
  */
 
 /*
- * @test id=z
+ * @test id=Z
```

Comment:
Are these id's actually case sensitive? Just curious.

Reply:
It seems like they are case insensitive. Just tried to problem list with z and Z, and both worked.